academy/ibc/migration.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "IBC Migration"
+order: 
+description: 
+tag: deep-dive
+---
+
+# IBC Migration
+
+When working with IBC, it is important to be able to upgrade IBC chains and clients. IBC-connected chains must perform an IBC upgrade if their upgrade will break counterparty IBC clients. This is of importance for all developers working with Tendermint clients enabling upgrades.
+
+The **IBC module** serves several different user groups, such as:
+
+* Chain operators
+* Relayer developers and operators 
+* IBC application and light client developers
+
+Each of these stakeholders has different expectations when it comes to **backward compatibility**. For example, the (IBC protocol specifications)[https://github.com/cosmos/ibc] and (IBC Go)[ibc-go] both maintain different versions and changes can be backward or non-backward compatible for different users.
+
+<HighlightBox type="info">
+
+When it comes to IBC Go there is an effort to ensure that all IBC Go releases do not hinder successful chain communication. All IBC Go releases are therefore using IBC protocol specification v1.0.
+
+</HighlightBox>
+
+All **IBC module releases** allow chains to communicate successfully with any chain running any version of the code. It is ensured that all major releases are supported by relayers so that they can relay between the new major release and older releases. Currently, an IBC protocol specification v2.0 upgrade is not planned because it would be very disruptive to the ecosystem.
+
+## Upgrading IBC clients - client breaking upgrades
```

Comment:
add link to this somewhere: https://github.com/cosmos/ibc-go/tree/main/docs/ibc/upgrades